frb_codegen/src/generator/dart/ty_boxed.rs

```diff
@@ -80,6 +83,8 @@ impl TypeDartGeneratorTrait for TypeBoxedGenerator<'_> {
             | Delegate(IrTypeDelegate::Array(_) | IrTypeDelegate::PrimitiveEnum { .. }) => {
                 format!("return _wire2api_{}(raw);", self.ir.inner.safe_ident())
             }
+            #[cfg(feature = "chrono")]
+            Delegate(IrTypeDelegate::Time(time)) => gen_wire2api_chrono(time),
```

Comment:
uber-nit: how is its friends? (e.g. date/datetime)

seems I have asked this question recently, not sure whether to you or someone else? If you have explained just point me to your old reply is enough ;)

Reply:
NaiveDate and Date and NaiveTime and Time you mean? They’re not yet supported

Reply:
I can create a Feature Issue for this and fix it at one point

Reply:
I see. Get it!